docs/managing-findings.md

```diff
@@ -75,32 +76,23 @@ These are hashed and returned as the syntactic identifier: `syntactic_id`. This
 
 ## Semgrep App
 
-Semgrep App builds on Semgrep CI findings to track state transitions and provide additional context for managing findings within your organization. Findings move between states according to their Semgrep CI `syntactic_id`, as mentioned above. A finding can occupy 4 states in Semgrep App: `OPEN`, `FIXED`, `MUTED`, `REMOVED`.
+Semgrep App builds on Semgrep CI findings to track state transitions and provide additional context for managing findings within your organization. A finding can occupy 4 states in Semgrep App: `OPEN`, `FIXED`, `IGNORED`, `REMOVED`.
 
 ### Finding states
 
-Semgrep App finding states are defined as follows:
+Semgrep App finding states are as follows:
 
-1. `OPEN`: the finding exists in the code and has not been muted.
-1. `FIXED`: the finding existed in the code, and is no longer found.
-1. `MUTED`: the finding has been ignored by a `nosemgrep` comment or via `.semgrepignore`.
-1. `REMOVED`: the finding's rule isn't enabled on the repository anymore. The rule was removed from the used ruleset, the rule was removed from the policy, or the containing policy was detached from the repo.
+1. `OPEN`: The finding exists in the code and has not been muted.
+1. `FIXED`: The finding existed in the code, and is no longer found.
+1. `IGNORED`: The finding has been ignored by a `nosemgrep` comment or through `.semgrepignore`.
+1. `REMOVED`: The finding's rule isn't enabled on the repository anymore. The rule was removed from the used ruleset, the rule was removed from the Rule Board.
```

Comment:
Please, check these statuses @chmccreery 🙏 , if they are still up to date. 

These statuses are also not visible to a user, right (like removed)? We are just trying to say what can happen to a finding (otherwise we are describing some internal logic that the user cannot see in the App, because removed findings just disappear from Findings page etc).

Reply:
Maybe I am confused from perspectives - triage states are something else than Semgrep App finding states...

Reply:
Open, Fixed, Ignored are all visible in the UI. With regards to "Removed" not sure how we present it lately. There is SOME overlap between triage states (action user undertakes) and under-the-hood states. In particular, "removed" is **not** a triage state - you did nothing to the finding itself, instead you did something to the rule that generated a finding.

Reply:
@s-santillan exactly. Since we are talking about what is happening under the hood, I am not sure this should be exposed to our users (the whole article is full of mentions about `syntactic_id` etc. this seems more like an article for people who would like to build our own FE for Semgrep). Please, @chmccreery, let me know what you think.

Reply:
I've tried to update these docs to remove syntactic_id because I agree that it exposes internals that don't belong in docs. I've also updated the fixed/ignored/open/removed module that Adam created for correctness.